shared/src/commonMain/kotlin/io/customer/shared/tracking/api/model/Metric.kt

```diff
@@ -0,0 +1,15 @@
+package io.customer.shared.tracking.api.model
+
+import io.customer.shared.tracking.constant.MetricEvent
+import kotlinx.serialization.SerialName
+
+/**
+ * Metric model to be used when making metrics tracking request.
+ */
+@kotlinx.serialization.Serializable
+internal data class Metric(
+    @SerialName("delivery_id") val deliveryID: String,
+    @SerialName("device_id") val deviceToken: String,
+    @SerialName("event") val event: MetricEvent,
+    @SerialName("timestamp") val timestamp: Long,
```

Comment:
```suggestion
    @SerialName("delivery_id") val deliveryID: String,
    @SerialName("device_id") val deviceToken: String,
    val event: MetricEvent,
    val timestamp: Long,
```

Assuming kotlinx serialization library doesn't require `@SerialName` unless the name is different from the property name, we only need to modify `deliveryID` and `deviceToken`.

Reply:
See my above response on `@SerialName`